### PR TITLE
Add banner to the top on the website

### PR DESCRIPTION
**Pull request by mdubus**
Addition of a banner allowing us to highlight a message

### CODE REVIEW

File: src/blocks/Header/index.js [hunk 1]
```diff
@@ -25,19 +35,15 @@ const Header = props => {
...
-          ? get('colors.valhalla.800')
-          : get('colors.valhalla'),
-      }}
-      data-cy="header"
-      {...props}
-    >
-      <DesktopHeader headerProps={headerProps} />
-      <MobileHeader headerProps={headerProps} />
+    <Wrapper data-cy="header" {...props}>
+      {banner && <Banner title={banner} />}
```
Comment (bidoubiwa) [inline src/blocks/Header/index.js:42]: Do you think it is worth adding the possibility to close the banner?

Comment (mdubus) [reply src/blocks/Header/index.js:42]: We thought about it and decided not to give this possibility :)

File: cypress/e2e/functional.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ describe(`Functional tests`, () => {
     })
 
     it('Should have a link bringing to home page', () => {
-      cy.get('a > span').contains('Bring me home, please').click()
+      cy.get('[data-cy="bring-me-home"]').click()
```
[the remark this answers is not in the document]
Comment (mdubus) [reply cypress/e2e/functional.js:103]: I had this test failing (that's a long story), and I decided to clean this a little bit to make it more easy to maintain ;)

**Comment by mdubus** [comment]
bors merge

**Pull request merged**